stdlib/public/core/MutableCollection.swift

```diff
@@ -121,7 +121,8 @@ where SubSequence: MutableCollection
   /// After partitioning a collection, there is a pivot index `p` where
   /// no element before `p` satisfies the `belongsInSecondPartition`
   /// predicate and every element at or after `p` satisfies
-  /// `belongsInSecondPartition`.
+  /// `belongsInSecondPartition`. This operation is not stable, so the
+  /// relative ordering of elements within the partitions is not preserved.
```

Comment:
Would it make sense to hedge this a bit, like we do with `sort`? 

```suggestion
  /// `belongsInSecondPartition`. This operation is not guaranteed to be
  /// stable, so the relative ordering of elements within the partitions may
  /// not be preserved.
```

(Here and elsewhere)

Reply:
How about:

```suggestion
  /// `belongsInSecondPartition`. This operation isn't guaranteed to be
  /// stable, so the relative ordering of elements within the partitions might
  /// change.
```

Avoiding "may" per reference guidelines, which can be ambiguous between possibility vs permission.  Also reframing it without the negation, which is often clearer.  Using a contraction, per DPSG, as I noted elsewhere.

Reply:
Updated! 👍🏻